doc/glm/second_level_model.rst

```diff
@@ -0,0 +1,123 @@
+press, which can be downloaded using :func:`nilearn.datasets.fetch_localizer_contrasts`::
+
+  from nilearn.datasets import fetch_localizer_contrasts
+  n_subjects = 16
+  data = fetch_localizer_contrasts(["left vs right button press"], n_subjects,
+                               get_tmaps=True)
+
+A first level model has been fit to this data and the contrasts maps for left-vs-right
+button press activation are available in the `cmaps` parameter. The second level model
+is fit using these contrast maps thus::
+
+  from nilearn.stats.second_level_model import make_second_level_design_matrix
+  second_level_input = data['cmaps']
+  design_matrix = pd.DataFrame([1] * len(second_level_input), columns=['intercept'])
+
+
+Two-sample testing
+------------------
+
+In two-sample testing, the activity attributed to two predictors is contrasted against each other, and so,
+two predictors need to be defined to fit the model. Consider the dataset that has activation corresponding
+to the presentation of horizonal and vertical checkerboards. This dataset can also be downloaded using
+:func:`nilearn.datasets.fetch_localizer_contrasts`::
+
+  from nilearn.datasets import fetch_localizer_contrasts
+  n_subject = 16
+  sample_vertical = fetch_localizer_contrasts(
+      ["vertical checkerboard"], n_subjects, get_tmaps=True)
+  sample_horizontal = fetch_localizer_contrasts(
+      ["horizontal checkerboard"], n_subjects, get_tmaps=True)
+  second_level_input = sample_vertical['cmaps'] + sample_horizontal['cmaps']
+
+Set up the design matrix::
+
+  import numpy as np
+  condition_effect = np.hstack(([1] * n_subjects, [- 1] * n_subjects))
+  subject_effect = np.vstack((np.eye(n_subjects), np.eye(n_subjects)))
+  subjects = ['S%02d' % i for i in range(1, n_subjects + 1)]
+  design_matrix = pd.DataFrame(
+      np.hstack((condition_effect[:, np.newaxis], subject_effect)),
+      columns=['vertical vs horizontal'] + subjects)
+
+As with first level models, the design matrix can be visualized using
+:func:`nilearn.reporting.plot_design_matrix()`
+
+.. image:: ../auto_examples/05_glm_second_level_models/images/sphx_glr_plot_second_level_two_sample_test_001.png
+   :target: ../auto_examples/05_glm_second_level_models/plot_second_level_two_sample_test.html#sphx-glr-auto-examples-05-glm-second-level-models-plot-second-level-two-sample-test-py
+
+
+Fitting a second level model
+============================
+
+Once the design matrix has been set up, fitting the model is simple. The class that
+provides tools for this is :class:`nilearn.stats.second_level_model.SecondLevelModel`.
+The :func:`nilearn.stats.second_level_model.SecondLevelModel.fit` function fits the
+model::
+
+  from nilearn.stats.second_level_model import SecondLevelModel
+  second_level_model = SecondLevelModel().fit(
+      second_level_input, design_matrix=design_matrix)
+
+
+While contrast maps can be computed using the
+:func:`nilearn.stats.second_level_model.compute_contrast` function. For instance, the
```

Comment:
The current CI error is because this target isn't defined. Looking [at the current master branch](https://github.com/nilearn/nilearn/blob/b5966b944f33e1abff9ab46e1ca450f35e95ed55/nilearn/stats/__init__.py#L4-L5), I think what you want here is:

```suggestion
:func:`nilearn.stats.contrasts.compute_contrast` function. For instance, the
```